data/Parameters_D.csv

```diff
@@ -0,0 +1,7 @@
+x1,x2,x3,x4,x5,x6,x7,x8
+0.05891223934526264,0.07729363212423296,0.13250621348370295,0.010813664989686791,0.024483796962931292,0.002480384884092555,0.0041040221876294915,0.03584929065117992
```

Comment:
I'd rather not store these in the repo, because they should be derived from the original data.